.github/workflows/generate.yml

```diff
@@ -0,0 +1,22 @@
+name: Generate
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Should we also add a scheduled run, e.g. once a day/week/?

Reply:
Added scheduled daily job.